packages/relay/test/report.test.ts

```diff
@@ -255,4 +261,131 @@ describe('POST /api/report', function () {
                 expect(res.body.error).to.be.equal('Invalid postId')
             })
     })
+
+    it('should fetch report whose reportEpoch is equal to currentEpoch - 1', async function () {
```

Comment:
Can we add one more test here is that, reportEpoch is equal to currentEpoch than empty list will return?

Reply:
👍🏻